libs/cart/state/src/reducers/cart-billing-address/cart-billing-address.reducer.ts

```diff
@@ -42,7 +42,7 @@ export function cartBillingAddressReducer<T extends DaffCart>(
           ...state.cart,
           billing_address: action.payload
         },
-        ...setLoading(state.loading, DaffLoadingState.Complete),
+        ...setLoading(state.loading, DaffStates.Complete),
```

Comment:
All of these completes should be stable instead, right? Are you going to do that in another PR?

Reply:
Yes. I'd like to leave this diff simply the transfer of DaffLoadingState to the equivalent DaffStates.